### ISSUE TITLE
tasks: catkin_build: build current package is not working anymore

### ISSUE THREAD

**Issue by FloMaile** [betwo/vscode-catkin-tools#39]
```bash
[build] Error: In order to use --this, the current directory must be part of a catkin package.
```

I debugged the build command and it turns out that the command is initiated in the workspace folder instead of the respective package folder e.g. `package_1`:

```bash
workspace_folder/        -- WORKSPACE
src/                               -- SOURCE SPACE
    package_1/
    package_N/
```

**Comment by betwo** [comment]
Hi @FloMaile, 
thanks for the feedback. That must be a relatively recent regression, I will look into the problem soon

**Comment by betwo** [comment]
Should be fixed in release 3.3.4. Please re-open this issue in case it somehow still does not work for you!
